src/javascript/DesignSystem/DatePicker/YearMonthSelector/YearMonthSelector.jsx

```diff
@@ -24,12 +24,30 @@ const style = () => {
     });
 };
 
-const YearMonthSelectorCmp = ({date, months, onChange, classes}) => {
+const calculateBeforeAfter = disabledDays => {
+    let before;
+    let after;
+
+    for (const day of disabledDays) {
```

Comment:
disabledDays can have two elements and it's weird structure. sometime [0] contains before sometimes contains after and the same for [1].

Reply:
ok let's keep it like that, do you think you can add comment to explain that on top of the function, so that future developer have informations about this structure